### ISSUE TITLE
Implement solo smartshots

### ISSUE THREAD

**Issue by fnoop** [fnoop/maverick#554]
Solo Smartshots have finally been opensourced.  Make the code run on generic platform, will probably need forked and ported to Maverick.

**Comment by terial** [comment]
Been trying to look into this. Is there a repository now?

**Comment by cglusky** [comment]
Think this is it...

https://github.com/OpenSolo/shotmanager

**Comment by fnoop** [comment]
@cglusky Yes that's it, hopefully when I get a spare moment and gain control of my inbox I'll have a stab at porting it to non-solo platform (so in the year 2050..)
